local-dev-first-start-only.sh

```diff
@@ -227,13 +227,18 @@ fi
 
```

Comment:
Can these changes be split out to their own script?  This is introducing vscode specific configuration.

Reply:
@sblack-usu and @Maurier I split this functionality into it's own repo here:
https://github.com/hydroshare/vscode

Added vscode directory to gitignore in #4486

Sorry to have bugged you with this Scott but hope @Maurier and others find it useful...